src/WebJobs.Script.WebHost/Helpers/MediaTypeMap.cs

```diff
@@ -28,11 +28,11 @@ public MediaTypeHeaderValue GetMediaType(string fileExtension)
             }
```

Comment:
dotnet 6 has `ArgumentNullException.ThrowIfNull` which prevents you from invoking throw. Having `throw` in a method prevents it from being inlined. So at least for `ArgumentNullException` I would suggest to use it when possible (like always). And for the other exceptions it might make sense to use a custom helper methods.

Reply:
Ooooh I indeed forgot this was added - thanks! I'll do a pass on the solution separately to this now that we're all .NET 6 here, SDK is trickier but there may be some places it benefits measurably.

Reply:
This is interesting! @NickCraver we can engage more folks here if that would be helpful.

Reply:
I was curious to see how much impact it'd have - minor as expected but this doesn't include bindings or the SDK which would increase effect - here's a PR moving to `ThrowIfNull` :) #7958